spark/v3.0/spark/src/main/java/org/apache/iceberg/spark/SparkReadConf.java

```diff
@@ -155,6 +155,18 @@ public long splitOpenFileCost() {
         .parse();
   }
 
+  /**
+   * For SparkFilesScan, determines the size of the target combined scan tasks. This allows
+   * for a different configuration than split size. Having this larger than split size allows files
+   * to be cut into parts and then recombined in an attempt to get closer to the target size.
+   */
+  public long planTargetSize() {
```

Comment:
Is it possible to use `targetSize` from the scan like we just updated #3400 to do?

Reply:
Nevermind, this looks different. Should this use the table property `write.target-file-size-bytes`?